integration-web/src/main/java/org/symphonyoss/integration/web/resource/ApplicationAuthenticationResource.java

```diff
@@ -206,4 +208,10 @@ private void validateRequiredParameter(String param, String paramName, String me
     }
   }
 
+  @ExceptionHandler(MissingRequiredParameterException.class)
```

Comment:
Please include this exception handler in the 'WebResourceExceptionHandler' component. We should avoid class exception handlers once we have a global exception handler.